### PR TITLE
Remove proposal description form field

### PR DESCRIPTION
**Pull request by beagleknight**
I have removed the field from the proposals form.
Proposal's summary max length is 350 now.

### CODE REVIEW

**Comment by josepjaume** [comment]
I've rebased to latest master @beagleknight

**Comment by beagleknight** [comment]
That's weird... I remember removing proposal's description from the proposals form :open_mouth:

File: app/views/proposals/_form.html.erb [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <div class="small-12 column">
       <%= f.label :summary, t("proposals.form.proposal_summary") %>
       <p class="note"><%= t("proposals.form.proposal_summary_note") %></p>
-      <%= f.text_area :summary, rows: 4, maxlength: 200, label: false,
+      <%= f.text_area :summary, rows: 4, maxlength: 350, label: false,
```
Comment (josepjaume) [inline app/views/proposals/_form.html.erb:21]: I guess summary is a `text` field on the db, right? In case not, do 350 characters fit?

Comment (beagleknight) [inline app/views/proposals/_form.html.erb:21]: Yeah it is. From pg manual a `text` field doesn't have a limit of characters (well, a field can contain at most 1GB)

Comment (josepjaume) [inline app/views/proposals/_form.html.erb:21]: :+1:

**Comment by josepjaume** [comment]
:+1: :tractor: :train: :train: :walking:

**Pull request merged**